src/main.ts

```diff
@@ -1,9 +1,20 @@
 import { NestFactory } from '@nestjs/core';
 import { ValidationPipe } from '@nestjs/common';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const options = new DocumentBuilder()
+    .addBearerAuth()
+    .setTitle('Nest blog')
+    .setDescription('The blog API description')
+    .setVersion('1.0')
+    .addTag('blog')
+    .build();
+  const document = SwaggerModule.createDocument(app, options);
+
+  SwaggerModule.setup('api', app, document);
```

Comment:
Usually 'api' prefix already use in the application. Better to use words like '_swagger_', _doc_, etc

Reply:
fixed